packages/web-components/test/agoric-wallet-connection.test.js

```diff
@@ -149,14 +147,14 @@ describe('AgoricWalletConnection', () => {
       );
     });
 
-    it('starts at locating', () => {
-      expect(el.state).to.equal('locating');
+    it('starts at locating', async () => {
+      await expect(el.state).to.equal('locating');
```

Comment:
could be expression but I suppose it's better to be consistent with the following tests and allow for multiple checks

Reply:
Ack